plutus-ledger/src/Ledger/Tx.hs

```diff
@@ -120,89 +134,116 @@ datumInDatumFromQuery _ DatumUnknown    = pure DatumUnknown
 datumInDatumFromQuery f (DatumInline d) = DatumInline <$> f d
 datumInDatumFromQuery f (DatumInBody d) = DatumInBody <$> f d
 
--- | Transaction output that comes from a chain index query.
---
--- It is defined here instead of the plutus-chain-index because plutus-ledger
--- uses that datatype, and plutus-ledger can't depend on plutus-chain-index
--- because of a cyclic dependency.
---
--- This datatype was created in order to be used in
--- 'Ledger.Constraints.processConstraint', specifically with the constraints
--- 'MustSpendPubKeyOutput' and 'MustSpendScriptOutput'.
-data ChainIndexTxOut =
-    PublicKeyChainIndexTxOut {
-      -- | Address of the transaction output. The address is protected by a
-      -- public key hash.
-      _ciTxOutAddress         :: Address,
+-- | Offchain view of a transaction output.
+data OffchainTxOut =
```

Comment:
Sorry for not saying it earlier...

May I suggest the initial name of the previous PR? `DecoratedTxOut`? My reasoning:
There is already `Cardano.Api.TxOut` which is an offchain `TxOut`. But the current `OffchainTxOut` is not exactly that. It is an offchain `TxOut` but with extra stuff inside. Hence why a name like `DecoractedTxOut`, `TxOutWithExtras`, `EnrichedTxOut`, etc. might be better.

Reply:
Let's go with `Decorated` then. If I'm fortunate enough my `sed` skills should be sufficient to change it quickly.